articles/azure-large-instances/what-is-azure-large-instances.md

```diff
@@ -0,0 +1,197 @@
+* Customer-coordinated maintenance windows
+* Application-aware snapshots, archive, mirroring, and cloning
+
+## SKU availability in Azure regions
+
+ALI offers multiple SKUs certified for specialized workloads.
+Use the workload-specific SKUs to meet your needs:
+
+* Large instances – two-socket to four-socket systems
+* Very Large instances – 4-socket to 20-socket systems
+
+ALI for specialized workloads is available in the following Azure regions:
+
+* West Europe
+* North Europe
+* Germany West Central zones support
+* East US zones support
+* East US 2 zones support
+* West US zones support
+* West US 2 zones support
+* South Central US
+* South Central US
+
+> [!Note]
+>Zones support refers to availability zones in which a region where ALI can be deployed across zones for high resiliency and availability. This capability enables support for multi-site active-active scaling.
+
+## Managing ALI in Azure
+
+Depending on your needs, the application topologies of Azure Large Instances can be complex.
+You may deploy multiple instances in one or more locations. The instances can have shared or dedicated storage, and specialized LAN and WAN connections
+Therefore, for ALI, Azure offers a consultation with a CSA/GBB in the field who can work with you.
+
+By the time your ALI implementation has been provisioned, the OS, networks, storage volumes, placements in zones and regions, and WAN connections between locations have already been configured.
+You're set to register your OS licenses (BYOL), configure the OS, and install the application layer.
+
+You'll see all the Azure Large instance resources, and their state and attributes, in the Azure portal.
+You can also operate the instances and open service requests and support tickets from there.
+
+## ALI stamp
+
+The Azure Large instance stamp itself combines the following components:
+
+* **Computing**
+Servers based on the generation of Intel Xeon processors that provide the necessary computing capability and are certified for the specialized workload.
+
+* **Network**
+A unified high-speed network fabric interconnects computing, storage, and LAN components.
+
+* **Storage**
+An infrastructure accessed through a unified network fabric.
+
+Within the multi-tenant infrastructure of the Azure Large instance stamp, customers are deployed in isolated tenants.
+When deploying a tenant, you name an Azure subscription within your Azure enrollment. 
+This Azure subscription is the one billed for your implementation of ALI.
+
+> [!Note]
+>A customer implementing ALI is isolated into a tenant.
+A tenant is isolated in the networking, storage, and compute layer from other tenants.
+Storage and compute units assigned to different tenants cannot see each other or communicate with each other on their implementations of ALI.
+
+## Operating system
+
+During the provisioning of the ALI, you can select the OS you want to install on the machines.
```

Comment:
May be rephrase this to indicate that only RHEL 8.4 is the available OS at this time @ianmsgit , @ryellajo12 . The current statement implies customers have a choice with OS for Epic but it isn't the case.

Reply:
Fixed 
To confirm in build